### DISCUSSION TITLE
Welcome to react-pdf-flipbook-viewer Discussions!

### DISCUSSION THREAD

**Discussion by mohitkumawat310** [mohitkumawat310/react-pdf-flipbook-viewer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by YashKharel19** [comment]
Hi, I found this very useful. How can I add audio and a page stacking feature (book-like) feature to this code?

**Reply by mohitkumawat310** [reply]
You can use events for audio; the page stacking feature is not available yet. However, I am working on a completely new implementation with Framer Motion that will have more features, smoother and faster animations, and easy implementation.
